transport-native-epoll/src/main/c/netty_epoll_native.c

```diff
@@ -394,14 +405,22 @@ static jstring netty_epoll_native_kernelVersion(JNIEnv* env, jclass clazz) {
     netty_unix_errors_throwRuntimeExceptionErrorNo(env, "uname() failed: ", errno);
     return NULL;
 }
-
 static jboolean netty_epoll_native_isSupportingSendmmsg(JNIEnv* env, jclass clazz) {
-    // Use & to avoid warnings with -Wtautological-pointer-compare when sendmmsg is
-    // not weakly defined.
-    if (&sendmmsg != NULL) {
-        return JNI_TRUE;
+    if (syscall(SYS_sendmmsg, -1, NULL, 0, 0) == -1) {
+        if (errno == ENOSYS) {
+            return JNI_FALSE;
+        }
     }
-    return JNI_FALSE;
+    return JNI_TRUE;
+}
```

Comment:
kind of nit suggestion but these methods could be more concise as e.g.
```c
return (syscall(SYS_sendmmsg, -1, NULL, 0, 0) != -1 || errno != ENOSYS) ? JNI_TRUE : JNI_FALSE;
```

Reply:
I think it is more readable as it is. So let me keep it